.github/workflows/release.yaml

```diff
@@ -0,0 +1,22 @@
+name: release
+
+on:
+  push:
+    tags:
+      - '**'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Extract release notes
+        uses: ffurrer2/extract-release-notes@v2
+        with:
+          release_notes_file: RELEASE_NOTES.md
```

Comment:
```suggestion
          release_notes_file: CHANGELOG.md
```

Reply:
`ffurrer2/extract-release-notes` reads `CHANGELOG.md` by default, see https://github.com/ffurrer2/extract-release-notes?tab=readme-ov-file#inputs.


And this might be conflicted with the [root changelog](https://github.com/olivierlacan/keep-a-changelog/blob/main/CHANGELOG.md).